backend/src/main/java/heartbeat/controller/report/ReportController.java

```diff
@@ -37,15 +38,15 @@ public class ReportController {
 	@Value("${callback.interval}")
 	private Integer interval;
 
-	@GetMapping("/{reportType}/{filename}")
+	@GetMapping("/{reportType}/{timeStamp}")
 	public InputStreamResource exportCSV(
 			@Schema(type = "string", allowableValues = { "metric", "pipeline", "board" },
 					accessMode = Schema.AccessMode.READ_ONLY) @PathVariable() ReportType reportType,
-			@PathVariable String filename) {
-		log.info("Start to export CSV file_reportType: {}, filename: {}", reportType.getValue(), filename);
-		InputStreamResource result = reportService.exportCsv(reportType, Long.parseLong(filename));
-		log.info("Successfully get CSV file_reportType: {}, filename: {}, _result: {}", reportType.getValue(), filename,
-				result);
+			@PathVariable String timeStamp, @Parameter String startDate, @Parameter String endDate) {
```

Comment:
put those 2 new variables into the API path

![image](https://github.com/au-heartbeat/Heartbeat/assets/8743692/3b362a0b-dc5d-4793-9503-4c441665e19d)

Reply:
done